#1 register jwt login

ParikhShaily03 opened the pull request:
Authentication error solving parameters are not appropiately passed

ParikhShaily03 committed:
authentication  service

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
AUthentication and token generation

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

juiTridhya-2025 commented on Controllers/Authcontroller.cs:
```diff
@@ -0,0 +1,82 @@
+﻿﻿using Microsoft.AspNetCore.Mvc;
```
why are we using Mvc in web api?

ParikhShaily03 replied:
we made controller class  which inherit controller functionality from controllaerbase class which provided by mvc 
also req/res and routing

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
authorise timeout modifiedMerge

ParikhShaily03 commented:
> Authentication error solving parameters are not appropiately passed

Okay

ParikhShaily03 committed:
Authentication jwt security

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
Port changed

ParikhShaily03 committed:
port chnagedPort changed

ParikhShaily03 committed:
commented user role

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
crud operation performing from authentication

ParikhShaily03 committed:
1. Migration
2.int  to dtring to department
3.addsecurity method
4.launch seting
5.usersecreet id added

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
deletion and new

ParikhShaily03 committed:
Merge branch '#1-Register-Jwt-Login' of https://github.com/ParikhShaily03/Student-Management-System into #1-Register-Jwt-Login

ParikhShaily03 committed:
All deleted and updated file comparing

ParikhShaily03 committed:
deleted file and.gitiignire
